Orange/evaluation/testing.py

```diff
@@ -366,21 +365,21 @@ def __init__(self, data, learners, store_data=False, store_models=False,
         self.call_callback(1)
 
 
-class Bootstrap(Results):
-    def __init__(self, data, learners, n_resamples=10, p=0.75, random_state=0,
-                 store_data=False, store_models=False, preprocessor=None,
-                 callback=None):
+class ShuffleSplit(Results):
+    def __init__(self, data, learners, n_resamples=10, p=0.75, test_size=None,
+                 random_state=0, store_data=False, store_models=False,
```

Comment:
The original scikit ShuffleSplit has train_size and test_size which can be fractions or int sizes, and either one of them or both can be provided. Our wrapper makes a mess of this by having parameters p and test_size with different defaults (providing only test_size uses p=0.75 instead of setting it to the complement).

I vote for sticking to scikit parameter names and defaults.